ignite/services/plugin/template/.gitignore.plush

```diff
@@ -18,6 +18,3 @@
 # Go workspace file
 go.work
 go.work.sum
-
-# App
-<%= Name %>*
```

Comment:
I think we should then exclude the build folder instead, otherwise you may commit the binary after an ignite chain build

Reply:
we are excluding the `*.ign` files

Reply:
I mean the chain binary